modules/planning/lattice/behavior_decider/adc_master_scenario.cc

```diff
@@ -0,0 +1,47 @@
+/***************************************************************************
+ *
```

Comment:
use apollo common header template

Reply:
Fixed the comment styles and removed outdated print messages.